src/main/java/com/tabnine/inline/TabnineDocumentListener.java

```diff
@@ -42,10 +39,9 @@ public void documentChangedNonBulk(@NotNull DocumentEvent event) {
         .invokeLater(
             () -> {
               int offset =
-                  editor.getCaretModel().getOffset()
-                      + (ApplicationManager.getApplication().isUnitTestMode()
-                          ? event.getNewLength()
-                          : 0);
+                  ApplicationManager.getApplication().isUnitTestMode()
+                      ? event.getOffset() + event.getNewLength()
+                      : editor.getCaretModel().getOffset();
```

Comment:
is this a bugfix? how did this affect us in tests and in production?

Reply:
I just changed the value in the test mode. the value in production stays the same.
it seems to be an issue with the previous value in the tests. 
Whoever wrote this line of code relies on the fact that `editor.getCaretModel().getOffset();` returns the offset after the doc change in the running mode, and in the test mode it returns the offset before the doc change.
I experienced some inconsistency with this value in the test mode in some cases (sometimes returns the old offset and sometimes the new offset), so I changed it to this one which should be consistent.